deploy/olm-catalog/windows-machine-config-operator/manifests/windows-machine-config-operator.clusterserviceversion.yaml

```diff
@@ -85,6 +86,20 @@ spec:
           - signers
           verbs:
           - approve
+        - apiGroups:
+          - machine.openshift.io
+          resources:
+          - machines
+          verbs:
+          - get
+          - list
+          - watch
+        - apiGroups:
+          - wmc.openshift.io
+          resources:
+          - '*'
+          verbs:
+          - '*'
```

Comment:
I still don't follow why it needs to be moved. Does it have something to do with `ClusterRole` vs `Role`?

Reply:
I think the problem is when we're define role which is specific to a namespace, the csv generated will have a section called `permissions` but it  is not clear in which namespace those permissions exist when multiple namespaces are being watched by the operator.

Reply:
`Role` can only be applied to a namespace. If you want to go across namespaces you need to use a `ClusterRole`. I don't think you can choose namespaces AFAIK. When it comes to the CSV it maps to `permissions` and `clusterPermissions`. So the reason for the movement is that what @PratikMahajan mentioned.

Reply:
A role will map to a namespace but you can define multiple roles and attach it to the same service account.The problem here is the operator when starting is trying to watch the `wmc` resource in `openshift-machine-api` namespace because of the way `watch` has been implemented in operator-sdk and since in csv we cannot mention `this is the namespace where I want the wmc CR to be watched, it is causing the problem`.

To be clear, the `wmc` CRD is still namespace scoped which should be accessible via a role but we're putting it in a clusterrole because the operator is trying to watch for all the objects in the all the namespaces that needs to watch even if we limit it to namespace.

Reply:
OK, we can stop this discussion. The question was why there was code movement and it has been answered.